src/app/services/utils.py

```diff
@@ -16,3 +22,28 @@ def resolve_bucket_key(file_name: str, bucket_folder: Optional[str] = None) -> s
         bucket_folder = cfg.BUCKET_MEDIA_FOLDER
 
     return f"{bucket_folder}/{file_name}" if isinstance(bucket_folder, str) else file_name
+
+
+def send_telegram_msg(chat_id: str, message: str, autodelete: bool = False) -> Optional[Response]:
+    """Send telegram message to the chat with the given id
+
+    Args:
+        chat_id (str): chat id
+        message (str): message to send
+        autodelete (str): delete msg after sending. Used for tests
```

Comment:
autodelete will be delete once the feature has been fully tested?

Reply:
It is actually a bool, not a string. This is used in unittests, that send a real msg (without notification) and delete it right after, not to pile-up. I tried to improve the docstring:

`autodelete (bool, default=False): delete msg after sending. Used by unittests`